app/create_file.py

```diff
@@ -1 +1,49 @@
-# write your code here
+import os
+from datetime import datetime
+import argparse
+
+
+def get_command_from_terminal() -> tuple:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-d", "--directories", nargs="*",
+                        help="creates directories -d dir1 dir2")
+    parser.add_argument("-f", "--filename", default="file.txt", required=False,
+                        help="creates file -f filename")
+    args = parser.parse_args()
+    directories = args.directories
+    filename = args.filename
+    return directories, filename
```

Comment:
not needed variables creation

```suggestion
    return args.directories, args.filename
```